src/libs/OptionsListUtils.js

```diff
@@ -104,9 +104,10 @@ function getPersonalDetailsForLogins(logins, personalDetails) {
     _.each(logins, (login) => {
         let personalDetail = personalDetails[login];
         if (!personalDetail) {
+            const displayName = Str.removeSMSDomain(login);
             personalDetail = {
                 login,
-                displayName: login,
+                displayName,
```

Comment:
```suggestion
                displayName: Str.removeSMSDomain(login),
```
Let's keep changes simple like this